doc/safety.md

```diff
@@ -0,0 +1,58 @@
+# SECURITY
+
+This document describes the security recommendations and issues that can be helpful for users of Bitcoin Core.
+
+ℹ️ There is no patch for human stupidity.
+
+Download Bitcoin Core only from below links, verify the integrity of the download before using:
+
+https://bitcoincore.org/en/download/
+
+https://github.com/bitcoin/bitcoin/releases
+
+Downloading Bitcoin Core from untrusted sources can result in using a [malware](https://bitcoin.stackexchange.com/a/107738/) that can steal your bitcoin and compromise your computer.
+
+### P2P
+
+Using non-default ports and non-listening nodes can improve security.
```

Comment:
This has a basis and you can read more about it in below links:

https://github.com/bitcoin/bitcoin/pull/23542

https://github.com/bitcoin/bitcoin/pull/23306

> Non-listening nodes have almost the exact same attack surface as listening ones

This is NOT true

Reply:
#23542 is currently in the state of an unmerged proposal.
Until this change in policy is merged, included in releases, and adopted by a large number of nodes (which will take time), it would be detrimental for the nework if many existing nodes witched to non-standard ports, because other nodes would avoid making outbound connections to them. Therefore, I don't think this suggestion should be included at this point in time.

Reply:
I've unresolved this.

As things are for now, and at least until #23542 is merged and in a release that is widely adopted on the network, using a non-default port is pretty much equivalent to disabling listening entirely. Outbound connections have a very strong bias away from such ports. That may change, but we shouldn't recommending it until the network actually has this policy adopted.

Regarding difference between listening and non-listening: non-listening nodes aren't as susceptible to eclipse attacks (which typically require making connections to the victim). There are also privacy differences, as controlling your outbound connections, and even just making fewer connections, reduces how much an attacker can infer about transactions propagating through you, and originating from you.

Reply:
> #23542 is currently in the state of an unmerged proposal.

I like the assumption that this PR will be merged before #23542 :)

Reply:
I don't assume that. But I also don't assume that this PR will only be merged in 2 years, when #23542 may be common on the network.

Reply:
At this point in time changing the port to one that is non default is the worst of both worlds; you both will receive absolutely no incoming traffic from legitimate nodes, any you do receive will be malicious / modified software.

Reply:
It is removed